.stylelintrc

```diff
@@ -23,12 +23,6 @@
           "custom-elements"
         ]
       }
-    ],
-    "selector-class-pattern": [
-      "^[a-z][a-zA-Z0-9]+$",
-      {
-        "message": "Expected class selector to be camelCase"
-      }
     ]
```

Comment:
It's a bit annoying that we can't comment out things here. Can we just disable stylelint for `/src/client/` instead, since it'll be removed in the `nextjs` branch soon anyway? And maybe for `src/app/(nextjs_migration)` as well.